src/components/biggive-main-menu/biggive-main-menu.tsx

```diff
@@ -73,6 +73,10 @@ export class BiggiveMainMenu {
     });
     this.setHeaderSize();
 
+    // repeat setting header size in attempt to workaround issue in COM-102 comments with the total raised box
+    // being hidden behind menu in some cases on Safari.
+    window.setTimeout(() => this.setHeaderSize(), 200);
```

Comment:
```suggestion
    window.setTimeout(() => this.setHeaderSize(), 1000);
```
I wonder if we would get an answer slightly quicker starting with a probably overly long time, even if the repaint is more visible, to be reasonably sure of the principle behind the fix/workaround first – then try reducing the time if it seems promising. Either way's definitely fine though.